scripts/hyperspectral/hyperspectral_workflow.sh

```diff
@@ -517,13 +518,13 @@ for ((fl_idx=0;fl_idx<${fl_nbr};fl_idx++)); do
 	typ_in_ENVI=$(grep '^data type' ${hdr_fl} | cut -d ' ' -f 4 | tr -d '\015')
 	xps_tm=$(grep 'current setting exposure' ${mtd_fl} | cut -d ':' -f 2 | tr -d '" ,\015' )
 	fl_clb="${drc_spt}/calibration_vnir_${xps_tm}ms.nc"
-	if [ "${wvl_nbr}" -eq 272 ]; then 
+	if [ "${wvl_nbr}" -eq 272 ] || [ "${wvl_nbr}" -eq 273 ]; then 
 	    flg_swir='Yes' # [flg] SWIR camera
 	elif [ "${wvl_nbr}" -eq 955 ]; then 
 	    flg_vnir='Yes' # [flg] VNIR camera
 	else
```

Comment:
Yes, if you want this to be fully generic then perhaps it should come from the sensor metadata file. note that the number of wavelengths is hardcoded in the calibration files (calibration_sensor_duration.nc files like calibration_vnir_30ms.nc). that does not yet apply to swir since we do not have swir calibration files yet. nevertheless i can change the workflow to detect the sensor type from the metadata file rather than the wvl_nbr. for posterity. but it means the initial workflow, which use to work with just raw+header files, will now require a third file (metadata) to complete.